src/exampleTypes/Facet.js

```diff
@@ -53,13 +53,37 @@ let SelectAll = observer(({ node, tree, Checkbox }) => {
     </label>
   )
 })
+
+let FacetOptionsFilter = ({ tree, node, TextInput, Button }) => {
+  let [val, setVal] = useState(node.optionsFilter)
+  let [buttonEnabled, setButtonEnabled] = useState(!!_.size(node.optionsFilter))
```

Comment:
buttonEnabled can just be a "computed" as `let buttonEnabled = val === node.optionsFilter`

Reply:
@daedalus28 good point